<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>10</y>
      <width>251</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="labelName2">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Player 2:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditName2">
       <property name="focusPolicy">
        <enum>Qt::ClickFocus</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labelName1">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Player 1:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditName1"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButtonReset">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>80</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>RESET</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonStart">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>80</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTurn">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>20</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Enter Player Names :</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>431</width>
      <height>430</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_2">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QPushButton" name="pushButton_22">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="text">
        <string>22</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pushButton_8">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item row="8" column="3">
      <widget class="QPushButton" name="pushButton_24">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="text">
        <string>24</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QPushButton" name="pushButton_21">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="text">
        <string>21</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="pushButton_11">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="text">
        <string>11</string>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QPushButton" name="pushButton_15">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="text">
        <string>15</string>
       </property>
      </widget>
     </item>
     <item row="8" column="2">
      <widget class="QPushButton" name="pushButton_23">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="text">
        <string>23</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_1">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QPushButton" name="pushButton_17">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="text">
        <string>17</string>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QPushButton" name="pushButton_19">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="text">
        <string>19</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QPushButton" name="pushButton_16">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="text">
        <string>16</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QPushButton" name="pushButton_18">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="text">
        <string>18</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="pushButton_4">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="pushButton_10">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="text">
        <string>10</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QPushButton" name="pushButton_14">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="text">
        <string>14</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButton_3">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="pushButton_9">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item row="6" column="4">
      <widget class="QPushButton" name="pushButton_20">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="text">
        <string>20</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="pushButton_12">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="text">
        <string>12</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButton_7">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="pushButton_5">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_6">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="pushButton_13">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="text">
        <string>13</string>
       </property>
      </widget>
     </item>
     <item row="8" column="4">
      <widget class="QPushButton" name="pushButton_25">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="text">
        <string>25</string>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>320</y>
      <width>491</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="1">
      <widget class="QLabel" name="labelWin">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>WIN</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="labelLose">
       <property name="text">
        <string>LOSE</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="labelTie">
       <property name="text">
        <string>TIE</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelPlayer1Record">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>Player 1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditWin1">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="lineEditLose1">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="lineEditTie1">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelPlayer2Record">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>player 2</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEditWin2">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="lineEditLose2">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLineEdit" name="lineEditTie2">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>170</y>
      <width>375</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="2" column="0">
      <widget class="QRadioButton" name="radioButtonYes">
       <property name="text">
        <string>YES</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QRadioButton" name="radioButtonNo">
       <property name="text">
        <string>No</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="labelPLayAgain">
       <property name="text">
        <string>Do you want to play again?</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuNew_Game">
    <property name="title">
     <string>New Game</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuExit">
    <property name="title">
     <string>Exit</string>
    </property>
   </widget>
   <addaction name="menuNew_Game"/>
   <addaction name="menuExit"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
